README.md

```diff
@@ -0,0 +1,63 @@
+# Chip8
```

Comment:
Nice job on the README @jrbetancourt98 It looks organized and it's super helpful to have all the build steps in one place!